src/arithmetic/numeric_funcs/numeric_funcs.c

```diff
@@ -172,4 +197,9 @@ void Register_NumericFuncs() {
 	types = array_append(types, (SI_NUMERIC | T_NULL));
 	func_desc = AR_FuncDescNew("sign", AR_SIGN, 1, 1, types, true);
 	AR_RegFunc(func_desc);
+
+	types = array_new(SIType, 1);
+	types = array_append(types, (SI_NUMERIC | T_STRING));
```

Comment:
With the change at 143, `SI_NUMERIC | T_STRING | T_NULL`

Reply:
👍